spanner/transaction.go

```diff
@@ -285,6 +294,7 @@ func (t *txReadOnly) ReadWithOptions(ctx context.Context, table string, keys Key
 					RequestOptions:      createRequestOptions(prio, requestTag, t.txOpts.TransactionTag),
 					DataBoostEnabled:    dataBoostEnabled,
 					DirectedReadOptions: directedReadOptions,
+					OrderBy:             orderBy,
```

Comment:
Shouldn't this be added to partitioned read interface as well? https://github.com/googleapis/google-cloud-go/blob/c15d69a8e71d1a7806e0b75bb6d62f499383a7ea/spanner/batch.go#L326

https://cloud.google.com/spanner/docs/samples/spanner-batch-client#spanner_batch_client-go

Reply:
This option does not applicable for partition reads, which is always act as if NO_ORDER was set.